node/src/bin/manager.rs

```diff
@@ -445,6 +442,36 @@ pub enum IndexCommand {
     },
 }
 
+#[derive(Clone, Debug, StructOpt)]
+pub struct FixBlockCommand {
+    /// Network name (must fit one of the chain)
```

Comment:
Maybe this should be called 'chain name', since you'd use `graphman chain list` to see which ones are valid. I would change teh help text to `Chain name (must be an existing chain, see 'chain list')`

Reply:
Fixed in 284ee30d4